packages/bitcore-node/test/integration/websocket.integration.ts

```diff
@@ -104,4 +112,59 @@ describe('Websockets', function() {
     expect(hasSeenTxEvent).to.equal(true);
     expect(hasSeenCoinEvent).to.equal(true);
   });
+
+  it('should get wallet events', async () => {
```

Comment:
please add a test for failed authentication: bad pub key, bug signature, missing signature.